ilastik/applets/neuralNetwork/nnClassApplet.py

```diff
@@ -44,7 +44,9 @@ def on_classifier_changed(slot, roi):
 
         self._topLevelOperator.classifier_cache.Output.notifyDirty(on_classifier_changed)
 
-        super(NNClassApplet, self).__init__("NN Training", workflow=workflow)
+        applet_name = ALLOW_TRAINING and "NN Training" or "NN Prediction"
```

Comment:
Nitpick, but I think this reads somewhat better:
```suggestion
        applet_name = "NN Training" if ALLOW_TRAINING else "NN Prediction"
```